charts/values.yaml

```diff
@@ -236,8 +241,13 @@ readinessProbe:
 
 ## Configure options for startup probe
 ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes
+
+################################################################
+## NOTE: See Limitations and known Issues in the README file  ##
+################################################################
+
 startupProbe:
-  enabled: true
+  enabled: false
```

Comment:
startupProbe should be fine to be enabled as when  ready file is present, it will pass.